src/pages/resultsView/ResultsViewPageStore.ts

```diff
@@ -760,6 +760,28 @@ export class ResultsViewPageStore {
         },
     });
 
+    @computed.struct get comparisonGroupsReferencedInURL() {
```

Comment:
hey, can you explain struct to me?  i looked it up.   it says "ignore new values that are structurally equal" but i don't quite understand what structurally equal means?

Reply:
somethings lost in translation there but it means deep equal

Reply:
got it.  thanks.   i guess struct comes from structurally equal